pom.ozone.xml

```diff
@@ -65,7 +65,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 
   <properties>
     <!-- required as child projects with different version can't use ${project.version} -->
-    <hadoop.version>3.3.0-SNAPSHOT</hadoop.version>
+    <hadoop.version>3.2.0</hadoop.version>
```

Comment:
this line can be even deleted AFAIK. There is no project which has this pom.ozone.xml as the parent. Therefore this hadoop.version property is not used at all. (but not  a problem to keep it).